examples/js/loaders/GLTFLoader.js

```diff
@@ -2145,7 +2145,7 @@ THREE.GLTFLoader = ( function () {
 
 			if ( isObjectURL === true ) {
 
-				URL.revokeObjectURL( sourceURI );
```

Comment:
Are you patching `URL.createObjectURL` to get it to work in node.js? Is there any reason `revokeObjectURL` shouldn't be patched in the same way?

Reply:
Yes, I think it can be patched in the same way.